### ISSUE TITLE
Property Sort Order not working inside media queries

### ISSUE THREAD

**Issue by DerFrenk** [brigade/scss-lint#236]
I think I've found a bug, the Property Sort Order check doesn't give any errors on incorrectly ordered code inside a media query block.

``` scss
.test {
  background-color: #fff;
  font-size: 12px;
}
```

This code is full valid.

When I switch the background-color and the font-size property, SCSS-lint will return this error (as expected)

```
Test.scss:2 [W] PropertySortOrder: Properties should be sorted in order, with
vendor-prefixed extensions before the standardized CSS property
```

Now switch back background-color and font-size, and add a new media query with incorrectly sorted properties:

``` scss
.test {
  background-color: #fff;
  font-size: 12px;

  @media screen and (min-width: 500px) {
    font-size: 12px;
    background-color: #fff;
  }
}
```

SCSS-lint doesn't give an error on the code above, but off course font-size should be positioned below background-color.

BTW, I noticed this bug while using a custom property sort order, so it doens't matter whether you're using the default or a custom property sort order.
